### PR TITLE
Feat/GitHub login test

### PR DESCRIPTION
**Pull request by pandey-shubh**
- added github login e2e

### CODE REVIEW

File: openlogin/login-with-github/index.config.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    user: { email: "connie.washington.1981@gmail.com" },
+  },
+  {
+    browserName: "firefox",
+    user: { email: "connie.washington.1981@gmail.com" },
+  },
+  {
+    browserName: "webkit",
+    user: { email: "connie.washington.1981@gmail.com" },
+  },
```
Comment (tainguyentt) [inline openlogin/login-with-github/index.config.ts:23]: my only concern is that we're using the same account for different browsers and our tests are running in parallel

**Review by tainguyentt — APPROVED** [review]
lgtm

File: .github/workflows/openlogin.login-with-github.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+name: OpenLogin - Github Login
+
+on:
+  schedule:
+    - cron: "*/15 * * * *"
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: 14
```
Comment (chaitanyapotti) [inline .github/workflows/openlogin.login-with-github.yml:9]: Move to node 16 everywhere

File: openlogin/login-with-github/index.config.ts (hunk 1 above)
Comment (chaitanyapotti) [reply openlogin/login-with-github/index.config.ts:23]: let's create a couple more users here pls

File: openlogin/login-with-github/index.lib.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import * as playwright from "@playwright/test";
+
+export interface TestArgs {
+  openloginURL: string;
+  user: { email: string };
+}
+
+export const test = playwright.test.extend<TestArgs>({
+  openloginURL: "https://app.openlogin.com",
```
Comment (chaitanyapotti) [inline openlogin/login-with-github/index.lib.ts:9]: Make this an env variable

File: openlogin/login-with-github/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  // Login with Github
+  await page.goto(openloginURL);
+  await page.click('button:has-text("Get Started")');
+  await page.click('[aria-label="Continue with existing GitHub"]');
+
+  try {
+    await page.waitForSelector("text=Reauthorization required", {
+      timeout: 10 * 1000,
+    });
+    await page.click('button:has-text("Authorize TorusLabs")');
```
Comment (chaitanyapotti) [inline openlogin/login-with-github/index.test.ts:15]: this text must've changed since then to `Authorize Web3Auth`

Comment (pandey-shubh) [reply openlogin/login-with-github/index.test.ts:15]: it's still showing : Authorize TorusLabs

File: .github/workflows/openlogin.login-with-github.yml (hunk 2 above)
Comment (pandey-shubh) [reply .github/workflows/openlogin.login-with-github.yml:9]: Updated

File: openlogin/login-with-github/index.config.ts (hunk 1 above)
Comment (pandey-shubh) [reply openlogin/login-with-github/index.config.ts:23]: added different github accounts for different browsers

File: openlogin/login-with-github/index.lib.ts (hunk 3 above)
Comment (pandey-shubh) [reply openlogin/login-with-github/index.lib.ts]: introduced an env variable PLATFROM which can be configured from github action file. possible values : PROD, STAGING

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**